model_analyzer/config/input/config_command.py

```diff
@@ -179,3 +179,9 @@ def __deepcopy__(self, memo):
 
     def __getattr__(self, name):
         return self._fields[name].value()
+
+    def __setattr__(self, name, value):
+        if name == '_fields':
+            self.__dict__[name] = value
+        else:
+            self._fields[name]._field_type._value = value
```

Comment:
is _fields[name] a ConfigValue or a ConfigField?  Regardless, I think they have a set_value() function that should be used here.

Reply:
Using set_value()